src/Traits/HandleUnauthorized.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\Permission\Traits;
+
+trait HandleUnauthorized
+{
+    public function responseUnauthorized()
```

Comment:
@AlexVanderbist, is the function name is good?

Reply:
Good for me, alternative would be `handleUnauthorized()` as it's still part of the middleware `handle` method but up to you!

Reply:
much better your idea @AlexVanderbist :+1: